x-pack/plugins/infra/public/store/local/log_filter/reducer.ts

```diff
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { reducerWithInitialState } from 'typescript-fsa-reducers';
+import { reducerWithInitialState } from 'typescript-fsa-reducers/dist';
```

Comment:
Can we stick to importing from the top level of third-party packages as their documentation recommends unless a there's a pressing reason to access internals?

Reply:
I don't know where these changes came from... It was either part of the weird merge bullshit or VS Code did it.